pwiz/analysis/spectrum_processing/SpectrumList_IonMobility.hpp

```diff
@@ -41,8 +41,8 @@ class PWIZ_API_DECL SpectrumList_IonMobility : public msdata::SpectrumListWrappe
     static bool accept(const msdata::SpectrumListPtr& inner);
     virtual msdata::SpectrumPtr spectrum(size_t index, bool getBinaryData = false) const;
 
-    // N.B this should agree with the enum IONMOBILITY_TYPE in pwiz_tools\BiblioSpec\src\BlibUtils.h
-    enum class IonMobilityUnits { none, drift_time_msec, inverse_reduced_ion_mobility_Vsec_per_cm2, compensation_V };
+    // N.B this order starting with none=0 should agree with the enum IONMOBILITY_TYPE in pwiz_tools\BiblioSpec\src\BlibUtils.h
+    enum class IonMobilityUnits {waters_sonar = -1, none, drift_time_msec, inverse_reduced_ion_mobility_Vsec_per_cm2, compensation_V };
```

Comment:
Not a big deal, but is a waters_sonar entry really needed here? If equipment is SONAR, then the reader doesn't need to check units since it will know it needs to handle that special SONAR array scheme...

Reply:
What it really points out is that "IonMobilityUnits" is really a misnomer - should be "AdditionalSeperationType" or somesuch.